test/e2e/install.go

```diff
@@ -59,9 +59,12 @@ func veleroInstall(ctx context.Context, cli, veleroImage string, resticHelperIma
 			return errors.New("No object store provider specified - must be specified when using kind as the cloud provider") // Gotta have an object store provider
 		}
 	}
-
+	version, err := getVeleroVersion(ctx, cli, true)
```

Comment:
Base on the comment https://github.com/vmware-tanzu/velero/pull/4141/files#r708764973 from @zubron , the cli may not use a semver version, so let's use the parameters `veleroVersion` and `upgradeFromVeleroVersion` to determine the version of velero